src/core/aws-client.ads

```diff
@@ -271,24 +273,25 @@ package AWS.Client is
       Attachments  : Attachment_List := Empty_Attachment_List;
       Headers      : Header_List     := Empty_Header_List;
       User_Agent   : String          := Default.User_Agent;
-      HTTP_Version : HTTP_Protocol   := HTTPv1)
+      HTTP_Version : HTTP_Protocol   := HTTP_Default)
       return Response.Data;
    --  Send to the server URL a POST request with Data
    --  Post will retry one time if it fails.
 
    function Upload
-     (URL        : String;
-      Filename   : String;
-      User       : String          := No_Data;
-      Pwd        : String          := No_Data;
-      Proxy      : String          := No_Data;
-      Proxy_User : String          := No_Data;
-      Proxy_Pwd  : String          := No_Data;
-      Timeouts   : Timeouts_Values := No_Timeout;
-      Headers    : Header_List     := Empty_Header_List;
-      Progress   : access procedure
-        (Total, Sent : Stream_Element_Offset) := null;
-      User_Agent : String          := Default.User_Agent)
+     (URL          : String;
+      Filename     : String;
+      User         : String          := No_Data;
+      Pwd          : String          := No_Data;
+      Proxy        : String          := No_Data;
+      Proxy_User   : String          := No_Data;
+      Proxy_Pwd    : String          := No_Data;
+      Timeouts     : Timeouts_Values := No_Timeout;
+      Headers      : Header_List     := Empty_Header_List;
+      Progress     : access procedure
+                       (Total, Sent : Stream_Element_Offset) := null;
+      User_Agent   : String          := Default.User_Agent;
+      HTTP_Version : HTTP_Protocol   := HTTP_Default)
```

Comment:
Please remove this option it is just not supported. And I would add a Pre condition on Upload with connection to check that it is HTTP/1 for now.

Reply:
Done